CMakeLists.txt

```diff
@@ -290,6 +290,9 @@ CMAKE_DEPENDENT_OPTION(
   "Slicer_BUILD_ITKPython" OFF)
 mark_as_superbuild(Slicer_INSTALL_ITKPython)
 
+option(Slicer_BUILD_PARAMETERSERIALIZER_SUPPORT "Build Slicer with parameter serializer support" OFF)
```

Comment:
There is  a vtkjsoncpp module that explicitly makes jsoncpp in VTK available externally. I've tried it in Slicer and it works (I just had to add #include "vtk_jsoncpp.h" and add vtkjsoncpp to linked libraries).

Exposing vnl from ITK might be a mistake (http://www.vtk.org/pipermail/vtk-developers/2010-January/022597.html), but it's a really useful one. Having vnl exposed from ITK is great and we use it in several projects.